From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Anthony Ramine <nox@nox.paris>
Date: Mon, 28 Oct 2019 23:32:44 +0100
Subject: [PATCH] Don't include binast

It's a Rust crate, so this prevents us from publishing on crates.io
when vendoring in another Rust crate, and it's used only during
development to generate other files that are checked in in the
repository.

diff --git a/js/src/frontend/moz.build b/js/src/frontend/moz.build
index e5e8f47f0f06..3639425faace 100644
--- a/js/src/frontend/moz.build
+++ b/js/src/frontend/moz.build
@@ -79,11 +79,6 @@ if CONFIG['JS_BUILD_BINAST']:
         'BinASTTokenReaderMultipart.cpp',
     ]
 
-    if CONFIG['JS_STANDALONE']:
-        DIRS += [
-            'binast'
-        ]
-
     # Instrument BinAST files for fuzzing as we have a fuzzing target for BinAST.
     if CONFIG['FUZZING_INTERFACES'] and CONFIG['LIBFUZZER']:
         include('/tools/fuzzing/libfuzzer-flags.mozbuild')
